app/helpers/application_helper.rb

```diff
@@ -335,4 +335,25 @@ def speaker_width(rooms)
     # speaker picture padding: 4px 2px; and we want the picture to be a circle
     speaker_height(rooms) - 4
   end
+
+  ##
+  # Checks if there is any event in the collection events that starts in the given date
+  #
+  # ====Returns
+  # * +True+ -> If there is any event for the given date
+  # * +False+ -> If there is not any event for the given date
+  def any_event_for_this_date?(events, date)
```

Comment:
I do know why helpers exist, thank you ;-) Why is this not a `Program` instance method?